src/index.html

```diff
@@ -1,12 +1,311 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="navigation">
+      <div class="navigation__container">
+        <a href="#home" class="navigation__logo">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="navigation__section">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a href="#" class="navigation__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navigation__item" data-qa="nav-hover">
+            <a href="#" class="navigation__link">
+              Laptops&nbsp;&&nbsp;computers
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#" class="navigation__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card">
+      <article class="card__container" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Mono block (MNED2UA-A)"
+          class="card__picture"
+        >
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars__stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+        <div class="card__price">
```

Comment:
don't forget about an empty line between _multiline_ siblings
```suggestion
        </div>
        
        <div class="card__price">
```